scss/upload/_layout.scss

```diff
@@ -1 +1,6 @@
-@import "~@progress/kendo-theme-default/scss/upload/layout";
+$dropzone-upload-status-icon-margin: $padding-y !default;
+$upload-status-margin: 0 !default;
+$file-name-margin: 0 !default;
+$invalid-icon-margin: $padding-x-sm / 2 !default;
```

Comment:
I would suggest that we prefix the variables with `upload-`, so that the scope is clear. I would also suggest using `offset` instead of `margin`, as it doesn't specify how the distance is implemented (can be via padding or absolute positioning, instead of requiring a margin).

    $upload-dropzone-status-icon-offset: $padding-y !default;
    $upload-status-offset: 0 !default;
    $upload-file-name-offset: 0 !default;
    $upload-invalid-icon-offset: $padding-x-sm / 2 !default;